Fabric.Identity.API/Extensions/UtilityExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Fabric.Identity.API.Extensions
+{
+    public static class UtilityExtensions
+    {
+        public static string FormatUrl(this string url)
```

Comment:
Minor comment but maybe change the name to something like EnsureTrailingSlash??

Reply:
Good call, I'll update that.